buffer_pool_manager_test has a minor problem

https://github.com/cmu-db/bustub/blob/caf086904238a0dfda6c4b3cd808e6a1bb2d3f21/test/buffer/buffer_pool_manager_test.cpp#L37

`sizeof(page0->GetData())` is 8 bytes on a 64-bit OS or 4 bytes on a 32-bit OS.

Possible compile error, if the following string having size larger than 8. 
Example with `"Hello World"`:
```bash
/tmp/bustub/test/buffer/buffer_pool_manager_test.cpp:22:6: error: ‘Hello World’ directive output truncated writing 11 bytes into a region of size 8 [-Werror=format-truncation=]
```

The buffer here is the page, so the buffer size is `PAGE_SIZE`.

Possible relevant: [How to find the 'sizeof' (a pointer pointing to an array)? - Stackoverflow](https://stackoverflow.com/questions/6751749/what-is-the-size-of-a-pointer)

--- a/test/buffer/buffer_pool_manager_test.cpp
+++ b/test/buffer/buffer_pool_manager_test.cpp
@@ -34,7 +34,7 @@ TEST(BufferPoolManagerTest, DISABLED_SampleTest) {
   EXPECT_EQ(0, page_id_temp);
 
   // Scenario: Once we have a page, we should be able to read and write content.
-  snprintf(page0->GetData(), sizeof(page0->GetData()), "Hello");
+  snprintf(page0->GetData(), PAGE_SIZE, "Hello");
   EXPECT_EQ(0, strcmp(page0->GetData(), "Hello"));
 
   // Scenario: We should be able to create new pages until we fill up the buffer pool.

